### PR TITLE
CI

### PR DESCRIPTION
**Pull request by ian-r-rose**
Further work on making Aqueduct more robust, and allowing us to catch problems before they arise. This lints the codebase on PRs and makes sure that all the DAGs are importable. It does *not* run any of the DAGs.

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+trigger:
+  - master
+jobs:
+  - job: 'test_airflow'
+    pool:
+      vmImage: 'Ubuntu-16.04'
```
Comment (hunterowens) [inline azure-pipelines.yml]: given that our prod machines are ec2-linux, perhaps use RHEL is possible?

File: dags/myla311/dag_311_retrieve_update.py [hunk 2]
```diff
@@ -18,14 +18,13 @@
...
-USERNAME = Variable.get("SOCRATA_USERNAME")
-PASSWORD = Variable.get("SOCRATA_PASSWORD")
-
 
 def retrieve_save_data(**kwargs):
     # Using Authenticated Client:
     # API Document: https://dev.socrata.com/foundry/data.lacity.org/aub4-z9pc
+    MY_APP_TOKEN = Variable.get("SOCRATA_APP_TOKEN")
+    USERNAME = Variable.get("SOCRATA_USERNAME")
+    PASSWORD = Variable.get("SOCRATA_PASSWORD")
```
Comment (hunterowens) [inline dags/myla311/dag_311_retrieve_update.py]: any reason why this needs to be moved?

File: azure-pipelines.yml (hunk 1 above)
Comment (ian-r-rose) [reply azure-pipelines.yml]: Good point, I'll give that a shot.

File: dags/myla311/dag_311_retrieve_update.py (hunk 2 above)
Comment (ian-r-rose) [reply dags/myla311/dag_311_retrieve_update.py]: If these variables are not defined in the airflow db, then testing the integrity of these dags fails. I wanted them to be importable without actually running the dag for CI purposes.

File: azure-pipelines.yml (hunk 1 above)
Comment (ian-r-rose) [reply azure-pipelines.yml]: Looks like RHEL is not one of the [available VMs](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops#use-a-microsoft-hosted-agent)

**Pull request merged**